sources/install.sh

```diff
@@ -1903,12 +1910,100 @@ function install_volatility2() {
   add-test-command "volatility --help"
 }
 
+function install_volatility3() {
+  colorecho "Installing volatility3"
+  python3 -m pipx install git+https://github.com/volatilityfoundation/volatility3
+  add-aliases volatility3
+  add-history volatility3
+  add-test-command "volatility3 --help"
+}
+
+function install_testdisk() {
+  colorecho "Installing testdisk"
+  fapt testdisk
+  add-history testdisk
+  add-test-command "testdisk --help"
+}
+
+function install_jadx() {
+  colorecho "Installing jadx"
+  git -C /opt/tools/ clone https://github.com/skylot/jadx.git
+  cd /opt/tools/jadx
+  ./gradlew dist
+  add-aliases jadx 
+  add-history jadx
+  add-test-command "jadx --help"
+}
+
+function install_fdisk() {
+  colorecho "Installing fdisk"
+  fapt fdisk
+  add-history fdisk
+  add-test-command "fdisk --help"
+}
+
+function install_sleuthkit() {
+  colorecho "Installing sleuthkit"
+  fapt sleuthkit
+  add-test-command "blkcalc --help"
+  add-test-command "blkcat --help"
+  add-test-command "blkls --help"
+  add-test-command "blkstat --help"
+  add-test-command "fcat --help"
+  add-test-command "ffind --help"
+  add-test-command "fiwalk --help"
+  add-test-command "fls --help"
+  add-test-command "fsstat --help"
+  add-test-command "hfind --help"
+  add-test-command "icat --help"
+  add-test-command "ifind --help"
+  add-test-command "ils --help"
+  add-test-command "img_cat --help"
+  add-test-command "img_stat --help"
+  add-test-command "istat --help"
+  add-test-command "jcat --help"
+  add-test-command "jls --help"
+  add-test-command "jpeg_extract --help"
+  add-test-command "mactime --help"
+  add-test-command "mmcat --help"
+  add-test-command "mmls --help"
+  add-test-command "mmstat --help"
+  add-test-command "sigfind --help"
+  add-test-command "sorter --help"
+  add-test-command "srch_strings --help"
+  add-test-command "tsk_comparedir --help"
+  add-test-command "tsk_gettimes --help"
+  add-test-command "tsk_loaddb --help"
+  add-test-command "tsk_recover --help"
```

Comment:
Fyi you don't necessarily have to define lots of test commands. The purpose of test commands is to make sure the tool or framework is installed properly. In this particular case we can keep them, but if one or two commands work, there's no reason the others wouldn't.